mailer.js

```diff
@@ -276,6 +291,10 @@ module.exports = function (log) {
       'X-Verify-Code': message.code
     }
 
+    if (this.sesConfigurationSet) {
+      headers[X_SES_MESSAGE_TAGS] = sesMessageTagsHeaderValue(templateName)
```

Comment:
Since this is being sent for all emails, we could move it into `Mailer.prototype.send`. Something like 

```javascript
    if (this.sesConfigurationSet) {
      // Note on SES Event Publishing: The X-SES-CONFIGURATION-SET and
      // X-SES-MESSAGE-TAGS email headers will be stripped by SES from the
      // actual outgoing email messages.
      emailConfig.headers[X_SES_CONFIGURATION_SET] = this.sesConfigurationSet
      emailConfig.headers[X_SES_MESSAGE_TAGS] = sesMessageTagsHeaderValue(message.template)
}
```